src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -146,54 +138,54 @@ function WorkspaceMembersPage(props) {
 
     useEffect(() => {
         validateSelection();
-    }, [props.preferredLocale, validateSelection]);
+    }, [preferredLocale, validateSelection]);
 
     useEffect(() => {
-        if (removeMembersConfirmModalVisible && !_.isEqual(accountIDs, prevAccountIDs)) {
+        if (removeMembersConfirmModalVisible && JSON.stringify(accountIDs.sort()) !== JSON.stringify(prevAccountIDs.sort())) {
```

Comment:
Maybe you can use isEqual from lodash

Reply:
Shouldn't we avoid lodash?

Reply:
We should, but there are some cases where its usage is justified, like this one. You can import it like `import lodashIsEqual from 'lodash/isEqual';` and use it here